src/taco-livereload/lib/utils/homePage/bower_components/q/.npmignore

```diff
@@ -0,0 +1,23 @@
+node_modules
```

Comment:
And should we be including the bower version of Q verbatim in our codebase? Wouldn't it be better to take a dependency on bower and acquire Q on demand, so we can take fixes without requiring a code change if possible?